.storybook/viewport.js

```diff
@@ -7,62 +7,91 @@ export default {
     styles: {
       width: '320px',
       height: '480px'
-    },
-    type: 'mobile'
+      },
+      type: 'mobile'
```

Comment:
Not a real change.

Reply:
My bad, didn't intend on making that change.

Reply:
@dhruvkb Done!